.github/workflows/beta.yaml

```diff
@@ -48,6 +54,7 @@ jobs:
               - conda-forge
         create-args: >-
           python=${{ matrix.python-version }}
+          rdkit
```

Comment:
This is necessary because of a recent change in which RDKit is treated as optional - so it's not in the base environment

Reply:
https://github.com/openforcefield/openff-interchange/pull/1321

Reply:
(not blocking) Oh neat, I didn't realize you could add deps here